### PR TITLE
Clarified startsWith behavior on $search

### PR DESCRIPTION
**Pull request by Licantrop0**
made more explicit the startsWith behavior in $search

### CODE REVIEW

File: concepts/search-query-parameter.md [hunk 1]
```diff
@@ -130,25 +130,26 @@ Azure AD resources and their relationships that derive from [directoryObject](/g
...
-`https://graph.microsoft.com/v1.0/groups/?$search="displayName:OneVideo"`
-
-This looks for all groups with display names that look like "OneVideo". `$search` can be used together with `$filter` as well. For example:
+> [!NOTE]
+> - After tokenization, the tokens are matched independently of the original casing, and they are matched in any order. For example, displayName `李四(David Li)` will match search strings such as `李四(David Li)`, `李四`, `David`, `Li`, `David)`, `(李四`, `Li 李`.
+> - The tokenized search support works only on the **displayName** and **description** fields. Any field of String type can be put in `$search`; fields other than **displayName** and **description** default to `$filter` `startswith` behavior.
 
-`https://graph.microsoft.com/v1.0/groups/?$filter=mailEnabled eq true&$search="displayName:OneVideo"`
+For example:  
+`https://graph.microsoft.com/v1.0/groups/?$search="displayName:OneVideo OR mail:onevideo"`
```
Comment (FaithOmbongi) [inline concepts/search-query-parameter.md:138]: Suggestion to change this query format to use the Try It experience. That way, **consistencyLevel** can also be explicitly defined to help customers avoid false fails when they try the query.

File: concepts/search-query-parameter.md [hunk 2]
```diff
@@ -130,25 +130,26 @@ Azure AD resources and their relationships that derive from [directoryObject](/g
...
+> - The tokenized search support works only on the **displayName** and **description** fields. Any field of String type can be put in `$search`; fields other than **displayName** and **description** default to `$filter` `startswith` behavior.
 
-`https://graph.microsoft.com/v1.0/groups/?$filter=mailEnabled eq true&$search="displayName:OneVideo"`
+For example:  
+`https://graph.microsoft.com/v1.0/groups/?$search="displayName:OneVideo OR mail:onevideo"`  
+This looks for all groups with display names that has `one` and `video` tokens, or mail starting with `onevideo`.  
 
-This looks for all mail-enabled groups with display names that look like "OneVideo". The results are restricted based on a logical conjunction (an "AND") of the `$filter` and the entire query in the `$search`. The search text is tokenized based on casing, but matches are performed in a case-insensitive manner. For example, "OneVideo" would be split into two input tokens "one" and "video", but matches properties insensitive to case.
+`$search` can be used together with `$filter` as well:  
+`https://graph.microsoft.com/v1.0/groups/?$filter=mailEnabled eq true&$search="displayName:OneVideo"`
```
Comment (FaithOmbongi) [inline concepts/search-query-parameter.md:142]: Same comment as above.

File: concepts/search-query-parameter.md [hunk 3]
```diff
@@ -130,25 +130,26 @@ Azure AD resources and their relationships that derive from [directoryObject](/g
...
+The results are restricted based on a logical conjunction (an "AND") of the `$filter` and the entire query in the `$search`.
 
 The syntax of search follows these rules:
 
 * Generic format: $search="clause1" \[AND \| OR\] "\[clauseX\]"\.
 * Any number of clauses is supported. Parentheses for precedence is also supported.
 * The syntax for each clause is: "\<property>:\<text to search>".
 * The property name must be specified in the clause. Any property that can be used in `$filter` can also be used inside `$search`. Depending on the property, the search behavior is either "search" or "startsWith" if search is not supported on the property.
-* The whole clause must be declared inside double quotes. If it contains double quotes or backslash, it should be escaped with a backslash. No other characters need to be escaped.
+* The whole clause must be declared inside double quotes. If it contains double quotes or backslash, it should be escaped with a backslash. All the other special characters must be urlEncoded.
```
Comment (FaithOmbongi) [inline concepts/search-query-parameter.md:152]:
```suggestion
* The whole clause must be declared inside double quotes. If it contains double quotes or backslash, it should be escaped with a backslash. All the other special characters must be URL encoded.
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @Licantrop0 for raising this PR to clarify `$search`. Approved with minor suggestions.

File: concepts/search-query-parameter.md (hunk 1 above)
Comment (Licantrop0) [reply concepts/search-query-parameter.md:138]: added `msgraph-interactive` tag

File: concepts/search-query-parameter.md (hunk 2 above)
Comment (Licantrop0) [reply concepts/search-query-parameter.md:142]: added `msgraph-interactive` tag

**Pull request merged**